dev/system-test/firestore.ts

```diff
@@ -208,123 +208,127 @@ describe('Firestore class', () => {
   });
 });
 
-describe('CollectionGroup class', () => {
-  const desiredPartitionCount = 3;
-  const documentCount = 2 * 128 + 127; // Minimum partition size is 128.
+// Skip partition query tests when running against the emulator
```

Comment:
As a general comment to this PR, please ensure that each special case for the emulator is documented with an inline comment explaining _why_. As it is currently written, this comment merely repeats what the next line is clearly doing and, therefore, does not add value. As an outside observer, it is not clear to me _why_ this test suite needs to be skipped when running against the emulator and that rationale is worthy of an inline comment. In other places, it's not clear to me _why_ the expected behavior is different on the emulator. Please update/add comments like this.

Reply:
Agreed.